gulpfile.js

```diff
@@ -2,7 +2,7 @@ var gulp = require('gulp');
 var eslint = require('gulp-eslint');
 
 gulp.task('eslint', function () {
-  return gulp.src(['src/**'])
+  return gulp.src(['src/**', 'docs/src/**/*.{js,jsx}'])
```

Comment:
It was trying to lint over all the txt files in the ```src/components/raw-code``` directory and wasn't liking any of it.